packages/ui/src/lib/theme/defaultTheme.ts

```diff
@@ -227,7 +227,6 @@ export default {
       indigo: `bg-indigo-200 text-indigo-1100 border border-indigo-700`,
       blue: `bg-blue-200 text-blue-1100 border border-blue-700`,
       green: `bg-green-200 text-green-1100 border border-green-700`,
-      grass: `bg-grass-200 text-grass-1100 border border-grass-700`,
```

Comment:
I don't remember making this change either - if this is related to some of the Radix color cleanup that I saw recently merged in, disregard 👍

Reply:
Yup that's me from earlier